models/Asset/Image/Thumbnail.php

```diff
@@ -162,7 +164,7 @@ public function __toString(): string
     private function addCacheBuster(string $path, array $options, Asset $asset): string
     {
         if (isset($options['cacheBuster']) && $options['cacheBuster']) {
-            if (!str_starts_with($path, 'http')) {
+            if (!str_starts_with($path, 'http') && !str_starts_with($path, '/cache-buster-')) {
```

Comment:
We could merge this into one big if like 
```suggestion
             if (
            isset($options['cacheBuster']) && 
            $options['cacheBuster'] && 
            !str_starts_with($path, 'http') && 
            !str_starts_with($path, '/cache-buster-')
        ) {
            $path = '/cache-buster-' . $asset->getVersionCount() . $path;
        }
```
or we could split it into multiple ifs with early returns?
WDYT?